src/ProgressRing.Forms.Plugin.Android/ProgressRing.Forms.Plugin.Android.csproj

```diff
@@ -1,5 +1,6 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\..\..\Agrin.UI\packages\Xamarin.Forms.2.5.0.280555\build\netstandard1.0\Xamarin.Forms.props" Condition="Exists('..\..\..\..\..\Agrin.UI\packages\Xamarin.Forms.2.5.0.280555\build\netstandard1.0\Xamarin.Forms.props')" />

```

Comment:
It would be cool if you could fix the path to the forms package here (a path containing your "Agrin.UI" in my vanilla library will probably cause problems later. In fact, I would like the path to stay the same :)